draft-terbu-sd-jwt-vc.md

```diff
@@ -147,6 +147,11 @@ SD-JWT-based Verifiable Credential (SD-JWT VC):
 SD-JWT-based Verifiable Presentation (SD-JWT VP):
 : A Verifiable Presentation encoded using the Presentation format defined in [@!I-D.ietf-oauth-selective-disclosure-jwt].
 
+Unsecured payload of an SD-JWT VC:
+: A JSON object containing all selective and non-selective disclosable claims
+of the SD-JWT VC. The unsecured payload acts as the input JSON object to issue
+an SD-JWT VC complying to this specification.
+
```

Comment:
I don't think we need to define this here if we only use it once. I suggest to move this down to the example section.

Reply:
We are using the term in two sections, that is why I defined the term. But we can also remove it in the next version of the document if we don't think it adds any value.